mne/bem.py

```diff
@@ -905,8 +906,7 @@ def get_fitting_dig(info, dig_kinds='auto', verbose=None):
 
     .. versionadded:: 0.14
     """
-    if not isinstance(info, Info):
-        raise TypeError('info must be an instance of Info not %s' % type(info))
+    _validate_type(info, Info, "info", "Info")
```

Comment:
This is going to be common, should we special-case `Info` so we can omit the last two args here if the second is passed as `"info"` (i.e., `_validate_type(info, "info")`?

Reply:
I thought about that, yes. In this PR already?

Reply:
Up to you

Reply:
Then I'll do it later :) I do better with smaller PRs.